doc/troubleshooting.md

```diff
@@ -83,4 +83,43 @@ To make it work, the cluster admin will need to delete the lock by running
+NAME                                    TYPE           CLUSTER-IP       EXTERNAL-IP        PORT(S)                          AGE
+kubernetes                              ClusterIP      10.100.200.1     <none>             443/TCP                          6d
+pravega-bookie-headless                 ClusterIP      None             <none>             3181/TCP                         6d
+pravega-pravega-controller              LoadBalancer   10.100.200.11    10.240.124.15...   10080:31391/TCP,9090:30301/TCP   6d
+pravega-pravega-segmentstore-0          LoadBalancer   10.100.200.59    10.240.124.15...   12345:30597/TCP                  6d
+pravega-pravega-segmentstore-1          LoadBalancer   10.100.200.42    10.240.124.15...   12345:30840/TCP                  6d
+pravega-pravega-segmentstore-2          LoadBalancer   10.100.200.83    10.240.124.15...   12345:31170/TCP                  6d
+pravega-pravega-segmentstore-headless   ClusterIP      None             <none>             12345/TCP                        6d
+pravega-zk-client                       ClusterIP      10.100.200.120   <none>             2181/TCP                         6d
+pravega-zk-headless                     ClusterIP      None             <none>             2888/TCP,3888/TCP                6d
+
+```
+
+However this problem has been solved in Kubectl client version `v1.11.x` onwards.
+
+```
+# kubectl get svc
+NAME                                    TYPE           CLUSTER-IP       EXTERNAL-IP                     PORT(S)                          AGE
+kubernetes                              ClusterIP      10.100.200.1     <none>                          443/TCP                          6d20h
+pravega-bookie-headless                 ClusterIP      None             <none>                          3181/TCP                         6d3h
+pravega-pravega-controller              LoadBalancer   10.100.200.11    10.240.124.155,100.64.112.185   10080:31391/TCP,9090:30301/TCP   6d3h
+pravega-pravega-segmentstore-0          LoadBalancer   10.100.200.59    10.240.124.156,100.64.112.185   12345:30597/TCP                  6d3h
+pravega-pravega-segmentstore-1          LoadBalancer   10.100.200.42    10.240.124.157,100.64.112.185   12345:30840/TCP                  6d3h
+pravega-pravega-segmentstore-2          LoadBalancer   10.100.200.83    10.240.124.158,100.64.112.185   12345:31170/TCP                  6d3h
+pravega-pravega-segmentstore-headless   ClusterIP      None             <none>                          12345/TCP                        6d3h
+pravega-zk-client                       ClusterIP      10.100.200.120   <none>                          2181/TCP                         6d3h
+pravega-zk-headless                     ClusterIP      None             <none>                          2888/TCP,3888/TCP                6d3h
+
+```
+
+Also, while using older kubectl client versions, external IP details can still be viewed by doing a `kubectl describe svc` on the exposed service.
```

Comment:
Please mention that in version 1.10.x, `kubectl describe svc` can be used to view the complete external IP for the concerned service and also show the output here.